rplugin/python3/denite/scantree.py

```diff
@@ -0,0 +1,48 @@
+# ============================================================================
+# FILE: scantree.py
+# AUTHOR: Jorge Rodrigues <skeept at gmail.com>
+# License: MIT license
+# ============================================================================
+
+import sys
+from os.path import basename, join
+from os import walk
+
+try:
+    from os import scandir
+
+    def scantree(path_name, skip_list=None):
+        """if scandir is available use it as it is faster than os.walk"""
+
+        if skip_list is None:
+            skip_list = ['.git', '.hg']
```

Comment:
This value is shared with `scantree` without `scandir` so I suggest you to define `DEFAULT_SKIP_LIST` on a module level to make two function compatible.